imap_processing/ultra/utils/ultra_l1_utils.py

```diff
@@ -40,7 +40,13 @@ def create_dataset(
             "epoch": data_dict["spin_start_time"],
         }
         default_dimension = "spin_number"
-
+    elif "healpix" in data_dict:
```

Comment:
For these if/elif/else, can you comment with the 3 different scenarios these dict keys indicate? E.g. if the spin number is present, we're building the extended spin table.